how-it-works.html

```diff
@@ -87,11 +87,12 @@
                             data-query="how-it-works.alts.segment-2.img" data-attribute="alt"
                             alt="USE YOUR TRUSTED DEVICES">
                     </div>
-                    <div class="col-lg-6 px-4 my-auto mt-5 py-lg-5">
-                        <h2><strong class="monserrat-bold text-uppercase"
+                    <div class="col-lg-6 px-4 my-lg-auto mt-lg-5 py-lg-5">
+                        <h1><strong class="monserrat-bold text-uppercase "
                                 data-query="how-it-works.segment-2.title"></strong>
-                        </h2>
-                        <p class="monserrat-medium mt-4 fs-6" data-query="how-it-works.segment-2.description"></p>
+                        </h1>
+                        <p class="monserrat-medium mt-4 fs-5 color-neutral pe-lg-4 text-medium "

```

Comment:
Please use Monserrat regular accodring to figma for all descriptions(1,2 and 3) on this page.